.circleci/config.yml

```diff
@@ -30,9 +30,86 @@ jobs:
           name: Update CodeClimate coverage report
           command: |
               ./cc-test-reporter after-build --exit-code $?
+  build-and-upload-image:
+    docker:
+      - image: docker:20.10.14-git
+    parameters:
+      app_name:
+        type: string
+        description: Name of the application to build
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install dependencies
+          command: |
+            apk add --no-cache py3-pip
```

Comment:
Just out of curiosity. Are we using alpine here? As far as i know, apk is the package installer of alpine.

Reply:
Yes, the docker:20.10.14-git image that we are using has alpine as the operating system